client/my-sites/domains/components/mapping-instructions/index.jsx

```diff
@@ -48,10 +188,10 @@ class DomainMappingInstructions extends React.Component {
 		);
 	}
 
-	renderARecordsMappingMessage() {
+	renderARecordsInstructionsWithHeader() {
 		const { translate } = this.props;
 
-		const advancedSetupUsingARecordsTitle = translate( 'Advanced setup using root A records' );
+		const advancedSetupUsingARecordsTitle = translate( 'Alternative setup using root A records' );
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.